src/parse.js

```diff
@@ -126,8 +126,23 @@ export const getTraverser = (cb = noop, opts = {}) => {
         state.opts = {
           ...state.opts,
           ...opts,
-          filename: state.file ? state.file.opts.filename : opts.filename,
         };
+
+        let filename = state.file ? state.file.opts.filename : opts.filename;
+
+        if (filename) {
+          switch (state.opts.filename) {
+            case 'relative':
+              filename = filename.replace(process.cwd(), '');
```

Comment:
Always relative! E.g. `src/components/SiteHeader.js`. Leading slash means a path is absolute, so we have to remove that too.